ui_framework/components/table/kui_table_cell_liner.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import { KuiTd } from './kui_table';
+
+export function KuiTableCellLiner({ children }) {
+  return <KuiTd><div className="kuiTableRowCell__liner">{children}</div></KuiTd>;
+}
+KuiTableCellLiner.propTypes = {
+  children: React.PropTypes.any
```

Comment:
`React.PropTypes.node` includes arrays of renderable things:

> Validates that a prop is anything that can be rendered: numbers, strings, elements or an array (or fragment) containing these types.

So I think it is the appropriate type for `children`.